distribution/lib/Standard/Base/0.0.0-dev/src/Data/Dictionary.enso

```diff
@@ -459,7 +459,7 @@ key_widget dict:Dictionary -> Widget =
 
 ## PRIVATE
 key_value_widget -> Widget =
-    fqn = Meta.get_qualified_type_name Pair . take (..Before_Last '.')
+    fqn = Pair.to Meta.Type . qualified_name . take (..Before_Last '.')
```

Comment:
btw. is there any benefit of using `x.to T` instead of `(x : T)`?

Reply:
- both would work
- I thought the `to` variant was easier to search and replace
- as it required no parenthesis
- however people can now use both variants

The most important thing is whether the `Meta.Type.from (that:Any)` conversion is acceptable? Assuming it is, when I got an approval.